src/common/snippets/include/snippets/op/perf_count.hpp

```diff
@@ -11,6 +11,66 @@
 
 namespace ov {
 namespace snippets {
+
+namespace op {
+class PerfCountEnd;
+} // namespace op
+
+namespace utils {
+
+/**
+ * @interface Dumper
+ * @brief Dumper for node debug properties
+ * @ingroup snippets
+ */
+class Dumper {
+public:
+    Dumper() = default;
+    Dumper(const Dumper&) = delete;
```

Comment:
Interesting 🤔 
Why do we delete default copy constructors here and in the derived classes?
We don't manage any resources using pointers, so the default-generated would work as expected.

Reply:
Debug leftovers, removed